data/json/mutations.json

```diff
@@ -5050,6 +5050,13 @@
         "prereqs" : ["ACIDPROOF"],
         "threshreq" : ["THRESH_INSECT", "THRESH_CHIMERA", "THRESH_MEDICAL", "THRESH_SLIME"],
         "category" : ["MUTCAT_INSECT", "MUTCAT_CHIMERA", "MUTCAT_MEDICAL", "MUTCAT_SLIME"]
+    },{
+        "type" : "mutation",
+        "id" : "PYROMANIA",
+        "name" : "Pyromania",
+        "starting_trait" : true,
+        "points" : 1,
+        "description" : "Fire has always enthralled and fascinated you, and you gain a mood boost from lighting and being near them."
```

Comment:
Suggest `you gain a mood boost from lighting and being near them.` -> `you gain a boost to mood from igniting and proximity to them.`